components/timezone/src/lib.rs

```diff
@@ -105,7 +106,9 @@ pub use ids::{
     TimeZoneIdMapperWithFastCanonicalizationBorrowed,
 };
 pub use provider::TimeZoneBcp47Id;
+pub use time_zone::models;
 pub use time_zone::TimeZoneInfo;
+pub use time_zone::TimeZoneModel;
```

Comment:
when does a user need access to this trait? I don't think we need it reexported.

Reply:
It shows up in a public API signature and I do not like to not export things that show up publicly. I'm happy to export it from the `scaffold` module..